### PR TITLE
[翻译完成] src/docs/release/breaking-changes/overlay-entry-rebuilds.md 

### LINKED ISSUE #615: No.168 - OverlayEntries 和 Routes 进行了重建优化
Markdown 文档：https://github.com/cfug/flutter.cn/blob/master/src/docs/release/breaking-changes/overlay-entry-rebuilds.md

文档预览地址：https://flutter.cn/docs/release/breaking-changes/overlay-entry-rebuilds

建议完成天数：3~5 天

### PR DESCRIPTION
**Pull request by Nayuta-D**
fix #615

### CODE REVIEW

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 1]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
 `OverlayEntry`s, and removes unnecessary rebuilds
 to improve performance.
 
+在此更改之前，当一个新的不透明 `OverlayEntry`（记作 A） 被添加到另一个 `OverlayEntry`（记作 B）上，或者 A 从 B 上移除时，
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:29]:
```suggestion
在此更改之前，当一个新的不透明 `OverlayEntry`（记作 A）被添加到另一个 `OverlayEntry`（记作 B）上，
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 2]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+B 将会重新构建。这些重建是不必要的，因为他们并不是关联了 A 并且由 A 的状态发生改变而触发的。
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:30]:
```suggestion
或者 A 从 B 上移除时，B 将会重新构建。
这些重建是不必要的，因为它们不是由 `OverlayEntry` 内部状态发生的改变而触发的。
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 3]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+这个破坏性的改动优化了我们对 `OverlayEntry` 进行添加和移除的场景，移除了不必要的重建，以此提高性能。
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:31]:
```suggestion
这个破坏性的改动优化了我们对 `OverlayEntry` 进行添加和移除的场景，
移除了不必要的重建以提高性能。
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 4]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:32]:
```suggestion
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 5]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+
 Since the `Navigator` internally puts each `Route` into an
 `OverlayEntry` this change also applies to `Route` transitions:
 If an opaque `Route` is pushed on top or removed from above another
 `Route`, the `Route`s below the opaque `Route`
 no longer rebuilds unnecessarily.
 
+由于 `Navigator` 在内部会把每一个 `Route` 嵌套在 `OverlayEntry` 中，因此这个改动同样作用于 `Route` 的变换：
+如果一个不透明的 `Route` 被添加到另一个 `Route` 的顶部或从顶部移除时，
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:41]:
```suggestion
如果一个不透明的 `Route` 被添加到栈顶，或是从另一个 `Route` 的上层被移除时，
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 6]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+位于该不透明的 `Route` 下面的 `Route` 将不再进行不必要的重建。
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:42]:
```suggestion
位于不透明的 `Route` 下面的 `Route` 将不再进行不必要的重建。
```

File: src/docs/release/breaking-changes/overlay-entry-rebuilds.md [hunk 7]
```diff
@@ -20,35 +26,61 @@ breaking change optimized how we handle the addition and removal of
...
+在大多数情况下，本次优化不需要你对代码进行任何更改。
+然而，如果你的应用错误地依赖了隐式重建，你可能会发现问题，
+这可以通过调用 `setState` 进行状态的变更来解决。
+
 Furthermore, this change slightly modified the shape of the
 widget tree: Prior to this change,
 the `OverlayEntry`s were wrapped in a `Stack` widget.
 The explicit `Stack` widget was removed from the widget hierarchy.
 
+此外，这一更改略微修改了 widget 树的形状：在此更改之前，`OverlayEntry` 集合嵌套在 `Stack` 中。
```
Comment (AlexV525) [inline src/docs/release/breaking-changes/overlay-entry-rebuilds.md:62]:
```suggestion
此外，这一更改略微调整了 widget 树的层级结构：
在此更改之前，`OverlayEntry` 集合嵌套在 `Stack` 中。
```

**Review by Nayuta-D — COMMENTED** [review]
辛苦A少，已修改

**Review by AlexV525 — APPROVED** [review]

**Pull request merged**